src/sentry/tasks/reports.py

```diff
@@ -361,7 +442,7 @@ def merge(target, other):
             partial(merge_sequences, function=safe_add),
         ),
         ("issue_summaries", prepare_project_issue_summaries, merge_sequences),
-        ("usage_summary", prepare_project_usage_summary, merge_sequences),
+        ("series_outcomes", prepare_project_usage_outcomes, merge_sequences),
```

Comment:
How dangerous is renaming this? since these are async tasks, maybe I can't do this because of inflight reports. cc @tkaemming

Reply:
This only gets run early on Monday mornings (like 5am PDT or something, I forget the exact time — it should be in the celerybeat schedule) so as long as you don't deploy around that time you should be ok. I don't think coming up with a migration plan for on-premise/single-tenant is worth it given the low probability of this being deployed at the time when it'd have any effect and the complexity of doing the migration. Good thinking ahead about this though.

Reply:
cc @BYK for viability.

Basically we generate reports for an orgs project, serialize it to JSON then send out a task for every user that adds a bit more to the report. But bc I'm changing this one key, in flight reports could theoretically break.

But I think Ted is right. Chances are low.

Reply:
Thanks for the heads up. We also delay the shutdown by extra seconds to make sure the queues are empty. Seems like Celery supports graceful shutdowns like this.